charactersheet/charactersheet/viewmodels/character/armor.js

```diff
@@ -90,6 +113,8 @@ function ArmorViewModel() {
             Utility.array.updateElement(self.armors(), self.currentEditItem(), self.editItemIndex);
         }
 
+        self.equipArmorHandler();
```

Comment:
That calls the method to detect if another armor or shield is equipped, and unequips it AFAIK.  It ensures only 1 armor and/or shield are equipped.